sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/specialized/BlobClientBaseJavaDocCodeSnippets.java

```diff
@@ -121,6 +123,16 @@ public void download() {
         // END: com.azure.storage.blob.specialized.BlobClientBase.download#OutputStream
     }
 
+    /**
+     * Code snippets for {@link BlobClientBase#downloadStreaming(OutputStream)}
+     */
+    public void downloadStreaming() {
+        // BEGIN: com.azure.storage.blob.specialized.BlobClientBase.downloadStreaming#OutputStream
+        client.downloadStreaming(new ByteArrayOutputStream());
+        System.out.println("Download completed.");
```

Comment:
Same question about printing the downloaded data

Reply:
not for this one. we don't assume that bytes are string in this example nor we want to hint customer that they should be using this api to handle text blobs.